Image_Assets/optimise_images.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#   optimise_images.py - Call convert on PNGs and JPEGs using Google's settings
+#   Requires: Python >= 3.6
+#   Requires: Dank memes in production.
+
+from subprocess import CalledProcessError, check_output  # noqa: S404
+from sys import argv, exit
+
+ARG = "--replace"
+_LEN = len(argv)
+FILENAME = "optimise_images.py"
+SEARCH_PATH = argv[1] if (_LEN > 1 and argv[1].lower() != f"{ARG}") else "."
+REPLACE = True if (ARG in [i.lower() for i in argv]) else False
+OPTIMISATION_MAP = {
+    ".jpg": ("-sampling-factor 4:2:0 -strip -quality 85 "
+             "-interlace JPEG -colorspace sRGB"),
+    ".png": "-strip"
+}
+
+
+if _LEN == 1:
+    print(f"{argv[0]} [directory with images] [--replace]")
+    print("If you supply --replace, then the existing images")
+    print("will be replaced, otherwise it will only be a dry run.")
+    print("Implied directories, e.g. '.' or '..' will also work.")
+    exit()
+
+
+def output(cmd):
+    try:
+        _ = check_output(f"{cmd} 2>/dev/null", shell=True)  # noqa: S602
+        return _.decode("utf-8").rstrip()
+    except(CalledProcessError) as CMD_ERROR:
```

Comment:
This is typically written as
```suggestion
    except CalledProcessError as CMD_ERROR:
```

The parentheses are used to denote a tuple of error classes but the spacing is usually left neutral.

If flake8 disagrees with me then go with it though.

Reply:
Good style point, flake8 does not seem to complain, so all clear!